test/suites/basedir.bash

```diff
@@ -4,7 +4,7 @@ SUITE_basedir_SETUP() {
     mkdir -p dir1/src dir1/include
     cat <<EOF >dir1/src/test.c
 #include <stdarg.h>
-#include <test.h>
+#include "test.h"
```

Comment:
Just so I understand, why is this needed? Isn't test.h included via the `-I` directive, not via `.`?

Reply:
WIll have to retry it on windows, see what the issue might have been (forgot)

Reply:
Doesn't seem to be needed anymore, if it ever was...